uffi-compat/uffi-compat.lisp

```diff
@@ -628,6 +632,12 @@ output to *trace-output*.  Returns the shell's exit code."
           `((with-foreign-strings ,(cdr bindings) ,@body))
           body)))
 
+#+ecl
+(defun foreign-string-length (foreign-string)
+  (declare (inline ffi:foreign-string-length))
+  (ffi:foreign-string-length foreign-string))
```

Comment:
It's so weird. Have you ever seen it in use? I was wondering if we should leave it unimplemented to maintain the same behavior across implementations. What do you think?